dev-java/tomcat-native/tomcat-native-1.2.17.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit java-pkg-2 java-ant-2
+
+DESCRIPTION="Allows Tomcat to use certain native resources for better performance"
+HOMEPAGE="https://tomcat.apache.org/"
+SRC_URI="mirror://apache/tomcat/tomcat-connectors/native/${PV}/source/${P}-src.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="static-libs test"
+
+RDEPEND="dev-libs/apr:1=
+	dev-libs/openssl:=
```

Comment:
I know you didn't change it but this should now be `dev-libs/openssl:0=` because SLOT 0.9.8 won't work here.

Reply:
Ok, done.